quadtree/src/lib.rs

```diff
@@ -246,4 +332,13 @@ mod tests {
             assert_eq!(&NodeId::from_str(id).unwrap().to_string(), id);
         }
     }
+
```

Comment:
Let's add one more test where we explicitly test where we explicitly test the output of `SpatialNodeId::from`? For example (I might have coordinates wrong :P)
```
assert_eq!(SpatialNodeId{level: 3, x:5, y:4}, SpatialNodeId::from(NodeId::from_str("r301").unwrap());
```

Reply:
Done, I had to switch x and y.